net-analyzer/openvas-scanner/openvas-scanner-22.7.3.ebuild

```diff
@@ -80,8 +82,7 @@ src_configure() {
 		"-DSYSCONFDIR=${EPREFIX}/etc"
 		"-DSBINDIR=${EPREFIX}/usr/bin"
 		"-DOPENVAS_FEED_LOCK_PATH=${EPREFIX}/var/lib/openvas/feed-update.lock"
-		"-DBUILD_WITH_SNMP=$(usex snmp)"
```

Comment:
This also removed BUILD_WITH_SNMP, is this intented?

Reply:
BUILD_WITH_SNMP is no longer used. Ago's tinderbox pointed we add 2 variable unused in build of 22.7.2:
One was a typo: -DPENVAS_RUN_DIR that should have been -DOPENVAS_RUN_DIR
and one was a legacy variable BUILD_WITH_SNMP (it was in 21.4.4-r2)
I cannot see the variable in the current package version upstream.
See: https://bugs.gentoo.org/909559